gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/webauthn_register.html

```diff
@@ -30,7 +30,7 @@
     <div class="section">
         <form class="form" role="form" id="register" method="post" th:action="${action}">
             <input type="hidden" id="username" name="username" th:value="${user.username}"/>
-            <div id="webauthn-error" style="display: none;">
+            <div id="webauthn-error" class="button error-text">
```

Comment:
removing this style should create problem as it reveals the span with`webauthn-register_failed` text
Keep the style and add the class as well. So it should be

<div id="webauthn-error" style="display: none;" class="button error-text">